app/api/sessions.py

```diff
@@ -104,12 +106,24 @@ def validate_date(self, data, original_data):
         type_='speaker')
 
 
-class SessionList(ResourceList):
+class SessionListPost(ResourceList):
     """
-    List and create Sessions
+    List Sessions
     """
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
```

Comment:
This is clearly a `403` error. The data is valid. The syntax is valid. But that specific user does not have _permission_ do it.